### ISSUE TITLE
Show transfer status with pv

### ISSUE THREAD

**Issue by quantumphaze** [digint/btrbk#42]
Have an option flag that pipes the transfer through the pv utility. pv monitors data going through a pipe, alowing things like counting the bytes transfered.

Should be something like
```
btrfs send $SRC | pv | btrfs receive $DEST
```
Note that pv has no way of knowing the total amount of data that needs to be tranfered since send/receive doesn't know, it just counts the bytes. (If I am mistakened and there is a way to determine the side of a snapshot then there is a flag to add that.)

**Comment by digint** [comment]
Thanks for the input, should be pretty easy to implement, and is definitvely useful when starting btrbk interactively.
The output looks promising:
```
# btrfs send /mnt/btr_pool/_btrbk_snap/root_gentoo.20150815 | pv > /dev/null
At subvol /mnt/btr_pool/_btrbk_snap/rootfs.20150815
317MiB 0:00:03 [577.2MiB/s] [   <=>                                                      ]
```
I guess I will implement a "-V" option switch for this.

**Comment by quantumphaze** [comment]
I would suguest a --progress flag similar to the equivalent flag in rsync because -V is often used as shorthand for --version

**Comment by digint** [comment]
Yes you're right. The problem is that btrbk does not support long ("--") arguments. On the other side, as btrbk grows I'll have to support this anyways some day.

**Comment by digint** [comment]
implemented "--progress" option in: d8e8df0ebd89f6be401e8f334d50e850f748e299

@quantumphaze feel free to reopen this issue if you have more suggestions!
